packages/flutter/lib/src/widgets/sliver_prototype_item_list.dart

```diff
@@ -0,0 +1,183 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+
+import 'basic.dart';
+import 'framework.dart';
+import 'sliver.dart';
+
+class _SliverPrototypeExtentListElement extends SliverMultiBoxAdaptorElement {
```

Comment:
let's put the public class first, to make this file's context easier to understand